### PR TITLE
chore(libs): Bump to latest versions

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [x] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/chart/src/Column.ts [hunk 1]
```diff
@@ -183,7 +183,7 @@ export class Column extends XYAxis {
                     ;
 
                 const domainLength = host.yAxisStacked() ? dataLen : columnScale.bandwidth();
-                columnGEnter.merge(columnGRect as any).each(function (this: SVGElement, d) {
+                columnGEnter.merge(columnGRect as any).each(function (this: SVGElement, d: any) {
```
Comment (jbrundage) [inline packages/chart/src/Column.ts]: Is this needed on every d3 `.each(` first argument?

File: packages/layout/src/Toolbar.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { select as d3Select } from "d3-selection";
 
 import "../src/Toolbar.css";
 
-export function Toolbar() {
+function Toolbar() {
```
Comment (jbrundage) [inline packages/layout/src/Toolbar.ts]: This should be in its own PR

File: packages/layout/src/Toolbar.ts [hunk 3]
```diff
@@ -96,3 +96,7 @@ Toolbar.prototype.exit = function (domNode, element) {
         w.target(null);
     });
 };
+
+export {
+    Toolbar
+};
```
[the remark this answers is not in the document]
Comment (jbrundage) [reply packages/layout/src/Toolbar.ts]: Disregard... I see that this is a cosmetic change now and that the export is unchanged

[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/layout/src/Toolbar.ts]: The export didn't change - what changed was "how" it was being exported (due to new syntax error in new typescript).

File: tests/test-chart/tsconfig.json [hunk 4]
```diff
@@ -14,7 +14,8 @@
     "lib": [
       "dom",
       "es5",
-      "es2015.promise"
+      "es2015.promise",
+"es2015.iterable"
```
Comment (jbrundage) [inline tests/test-chart/tsconfig.json]: Indentation

File: packages/layout/src/Toolbar.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (GordonSmith) [reply packages/layout/src/Toolbar.ts]: Yes - typescript's generated code is subtly different hence the change

File: packages/chart/src/Column.ts (hunk 1 above)
Comment (GordonSmith) [reply packages/chart/src/Column.ts]: Just when the compiler complains about it...
